hack/generate-everything.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -o errexit -o nounset -o pipefail -o xtrace
+
+SCRIPTDIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+cd "$SCRIPTDIR/.."
```

Comment:
I am wondering whether this could be simplified like this?
```diff
--- a/hack/generate-everything.sh
+++ b/hack/generate-everything.sh
@@ -1,8 +1,7 @@
 #!/bin/bash
 set -o errexit -o nounset -o pipefail -o xtrace

-SCRIPTDIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
-cd "$SCRIPTDIR/.."
+cd "$(dirname "${BASH_SOURCE[0]}")/.."

 # These 3 need to run in this order. Not for any logical reasons, but simply
 # because of the current state of dependence between the generated tasks
```